Geometry/TrackerNumberingBuilder/src/TrackerShapeToBounds.cc

```diff
@@ -29,25 +29,21 @@
   3. combine the global rotation from DDD with the rotation defined in 2.   
   */
 
-Bounds* TrackerShapeToBounds::buildBounds(const DDSolidShape& _shape, const std::vector<double>& _par) const {
+Bounds* TrackerShapeToBounds::buildBounds(const cms::DDSolidShape& _shape, const std::vector<double>& _par) const {
```

Comment:
@cvuosalo Do we also need to have two versions of this function (ie being able to take a DDSolidShape or cms::DDsolidShape as argument?) it's possible of course, but I think then when it gets used in GeometricDet::bounds(), we still need to make a choice for which one to use. I think that needs to be done explicitly (as is the case now). We could let GeometricDet::bounds() take an argument to switch between the two, but it seems more trouble than it's worth for what this function is doing (as it only uses the shape type to switch, but nothing else). But maybe I'm missing something...

Reply:
@adewit It should be OK to have just this version. The old `DDSolidShape` can be safely cast to the new one for the cases we use.